doc/dash/dash-sonic-hld.md

```diff
@@ -537,8 +540,9 @@ overlay_sip              = ip_address                ; overlay ipv6 src ip if ro
 overlay_dip              = ip_address                ; overlay ipv6 dst ip if routing_type is {servicetunnel}, transform last 32 bits from packet (dst ip) 
```

Comment:
Should this be changed to `overlay_dip_prefix`?

Reply:
Good find. Addressed